frontend/src/Charts/Chart.js

```diff
@@ -31,7 +31,7 @@ function Chart() {
         const response = await axios.get(
           `/api/stations/${station}/${typeGroup}?start=${dateFormat(
             dateState[0].startDate
-          )}&end=${dateFormat(dateState[0].endDate)}&type=${dataType}`
+          )}&end=${dateFormat(dateState[0].endDate)} 23:50&type=${dataType}`
```

Comment:
Ez miért 23:50?

Reply:
Feltetelezem a 10perces adatok miatt az utolso napi adat 23:50-es, bar azt ellenorizzetek, hogy nem-e exclusive a query, (a range-es lekeresek a vegukon jellemzoen azok) es a 23:50-es adat is kikuldesre kerul.